sympy/core/tests/test_args.py

```diff
@@ -682,6 +682,7 @@ def test_sympy__core__function__WildFunction():
 def test_sympy__core__mod__Mod():
     from sympy.core.mod import Mod
     assert _test_args(Mod(x, 2))
+    assert _test_args(Mod(3*x, 2))
```

Comment:
This test would pass without the changes here. This does not really test anything useful and should be removed.

Tests should be added in `sympy/core/tests/test_arit.py`.